ext/nnstreamer/tensor_filter/tensor_filter_llama2.cc

```diff
@@ -147,7 +150,21 @@ TensorFilterLlama2c::configure_instance (const GstTensorFilterProperties *prop)
 void
 TensorFilterLlama2c::invoke (const GstTensorMemory *input, GstTensorMemory *output)
 {
-  char *result, *prompt;
+  UNUSED (input);
+  UNUSED (output);
+
+  throw std::runtime_error (
+      "llama2c only supports invoke-dynamic mode. Set `invoke-dynamic=true`");
+}
+
+/**
+ * @brief Invoke llama2c using input tensors
+ */
+void
+TensorFilterLlama2c::invoke_dynamic (GstTensorFilterProperties *prop,
+    const GstTensorMemory *input, GstTensorMemory *output)
+{
+  char *result = nullptr, *prompt;
```

Comment:
Be careful. ```prompt``` is also a class property.
Using same name for a local var and a class property is not good.